client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -54,12 +61,25 @@ export default function NotificationSettings( { onClose, site, settings }: Props
 		}
 	}, [ settings?.monitor_notify_users_emails ] );
 
+	useEffect( () => {
+		if ( enableMobileNotification || enableEmailNotification ) {
+			setValidationError( '' );
+		}
+	}, [ enableMobileNotification, enableEmailNotification ] );
+
+	function handleAddEmail( recipients: Array< string > ) {
+		if ( recipients.length ) {
+			setValidationError( '' );
+		}
+		setAddedEmailAddresses( recipients );
+	}
+
 	const addEmailsContent = enableEmailNotification && (
 		<div className="notification-settings__email-container">
 			<TokenField
 				placeholder="Enter Email IDs"
```

Comment:
Just a minor note here. Could we change it to "Enter email address"? Should this use the translate function?

Reply:
Yes, I missed this. Updated here - https://github.com/Automattic/wp-calypso/pull/71479/commits/afe6d08ed3b2a5c8b5995231df323806ba804ec9